tests/phpunit/elementor/modules/atomic-widgets/test-atomic-button.php

```diff
@@ -13,6 +13,30 @@ class Test_Atomic_Button extends Elementor_Test_Base {
 		'widgetType' => 'a-button',
 	];
 
+	const MOCK_LINK = [
```

Comment:
Consider remove the `setUp` function and move `$this->instance = Plugin::$instance->elements_manager->create_element_instance( self::MOCK );` into the first test.

Reply:
Updated.